subsys/testsuite/ztest/include/zephyr/ztest_test_new.h

```diff
@@ -133,7 +147,7 @@ extern struct ztest_suite_node _ztest_suite_node_list_end[];
  * @param teardown_fn The function to call after running all the tests in this suite
  */
 #define ZTEST_SUITE(SUITE_NAME, PREDICATE, setup_fn, before_fn, after_fn, teardown_fn)             \
-	struct ztest_suite_stats UTIL_CAT(z_ztest_test_node_stats_, SUITE_NAME);                   \
+	struct ztest_suite_stats UTIL_CAT(z_ztest_suite_node_stats_, SUITE_NAME);  \
```

Comment:
nit: please run clang-format

Reply:
Will do. Thanks for the reminding.

Reply:
I tried `clang-format-12`. It made a few changes. Some doesn't make sense to me. Such as below one.
So I only adopt the change to above line 150.
```
.after = (after_fn), .teardown = (teardown_fn),   
```